node/src/Pos/Explorer/Txp/Local.hs

```diff
@@ -164,7 +164,7 @@ eTxProcessTransaction itw@(txId, TxAux {taTx = UnsafeTx {..}}) = do
                 action = eProcessTx curEpoch tx (TxExtra Nothing curTime txUndo)
                 -- NE.fromList is safe here, because if `resolved` is empty, `processTx`
                 -- wouldn't save extra value, thus wouldn't reduce it to NF
```

Comment:
AFAIU, now we also rely on that `verifyAllIsKnown` is `True` when we verify local transactions. It would be good to write about it I think.